lib/readline.js

```diff
@@ -920,6 +921,10 @@ function emitKeypressEvents(stream) {
       var r = stream[KEYPRESS_DECODER].write(b);
       if (r) {
         for (var i = 0; i < r.length; i++) {
+          if (r[i] === '\t' && typeof r[i + 1] === 'string' && iface) {
+            iface.isCompletionEnabled = false;
```

Comment:
This will throw if `iface` is undefined though. Need to:

a. Document it
b. Ignore it if `iface` is undefined